### ISSUE TITLE
Usage is vague for options supporting multiple values

### ISSUE THREAD

**Issue by polarathene** [lycheeverse/lychee#667]
Snippet from [README - CLI parameters](https://github.com/lycheeverse/lychee/blob/487d88cefe59040c6766d5fb9e8ead2041f6ea4f/README.md#commandline-parameters)

```
OPTIONS:
    -a, --accept <accept>                      Comma-separated list of accepted status codes for valid links
    -b, --base <base>                          Base URL or website root directory to check relative URLs e.g.
                                               https://example.com or `/path/to/public`
        --exclude <exclude>...                 Exclude URLs from checking (supports regex)
    -s, --scheme <scheme>...                   Only test links with the given schemes (e.g. http and https)
```

I only just realized the syntax is implied with the `...` when raising this issue. It didn't stand out to me when looking at the README :sweat_smile: 

---

The `--scheme` option implied multiple values could be supplied, and it did not click for me when I had to separate the input with `--` after trying the option (_NOTE: I've seen discussion for implicit behaviour when no inputs are provided, which would have prevented the helpful error_).

When trying to use this, I originally tried `--scheme 'file://'` (didn't work), then after realizing to drop `://` suffix, I tried `--scheme 'http file'`, `--scheme 'http,file'` (_`--accept` mentions a comma separated list_), and `--scheme 'http|file'` (_`--exclude` had an example after the listed options using regex `|` pattern_).

I was not successful until `--scheme 'http' --scheme 'file'`. It did not occur to me that providing multiple values without wrapping them together in quotes was how it was probably intended (`--scheme 'http' 'file'` / `--scheme http file`).

Some brief examples may have been useful? It would also be helpful to clarify that these options are for links in inputs, originally I thought they also applied to inputs (_as redundant as that may seem, I was trying to [troubleshoot why my local inputs were attempting to be requested remotely](https://github.com/lycheeverse/lychee/issues/660), thinking `--offline` would apply as well_).

**Comment by mre** [comment]
We should definitely add usage examples for that yeah!

There will be a proper lychee book with examples at some point, but in the meantime we could add it to the `README.md` (and maybe the troubleshooting guide?).
We could even start a folder with example snippets for different use-cases that we can later put on into the book.
Open for suggestions and pull requests here.

**Comment by mre** [comment]
At long last we got proper documentation for repeating options in the [lychee docs here](https://lychee.cli.rs/#/usage/cli?id=repeating-options). 🥳 There is a button to edit the source in case we need to make any adjustments, but it's a start. 

(P.S.: There is also a dark mode for the docs now. 🤫)

Closing this issue because there is a "fix" now, but feel free to post a comment below to let me know what you think!
